task/validate-fbc/0.1/validate-fbc.yaml

```diff
@@ -216,18 +216,26 @@ spec:
             - SETFCAP
       computeResources:
         limits:
-          memory: 4Gi
+          memory: 8Gi
         requests:
-          memory: 512Mi
+          memory: 8Gi
```

Comment:
This is a large memory increase, likely because of the use of `jq --slurp` in the konflux-test image. I think that we should look to use something other than `jq` to determine the unique related images between the fragment and the index. We already have the content in files: 
https://github.com/konflux-ci/konflux-test/blob/d28ed88a75b49132cf734ba3ed4e1013b8696492/test/utils.sh#L637-L640

So we should be able to do the set subtraction in bash by sorting the files and then using `comm`: https://stackoverflow.com/a/11964477